graylog2-web-interface/webpack.config.js

```diff
@@ -14,7 +14,12 @@ const TARGET = process.env.npm_lifecycle_event;
 process.env.BABEL_ENV = TARGET;
 
 const BABELRC = path.resolve(ROOT_PATH, '.babelrc');
-const BABELLOADER = 'babel-loader?cacheDirectory&extends=' + BABELRC;
+const BABELOPTIONS = {
```

Comment:
Just to not forget about it: in order to use [tree shaking](https://webpack.js.org/guides/tree-shaking/) we need to update the [`babelrc`](https://github.com/Graylog2/graylog2-server/blob/master/graylog2-web-interface/.babelrc) file to not transform ES2015 modules:

```javascript
{
  presets: [
    ["es2015", { "modules": false }]
  ]
}
```

Source: https://babeljs.io/docs/plugins/preset-es2015/